diff -Naur sonnet-5.11.0/src/plugins/hunspell/hunspellclient.cpp sonnet-5.11.0.tpg/src/plugins/hunspell/hunspellclient.cpp
--- sonnet-5.11.0/src/plugins/hunspell/hunspellclient.cpp	2015-06-06 21:32:56.000000000 +0000
+++ sonnet-5.11.0.tpg/src/plugins/hunspell/hunspellclient.cpp	2015-06-14 15:13:08.327313462 +0000
@@ -47,7 +47,7 @@
 QStringList HunspellClient::languages() const
 {
     QStringList lst;
-    QDir dir(QStringLiteral("/usr/share/myspell/dicts/"));
+    QDir dir(QStringLiteral("/usr/share/dict/ooo/"));
     if (dir.exists()) {
         QStringList lstDic = dir.entryList(QStringList(QStringLiteral("*.dic")), QDir::Files);
         foreach (const QString &tmp, lstDic) {
diff -Naur sonnet-5.11.0/src/plugins/hunspell/hunspelldict.cpp sonnet-5.11.0.tpg/src/plugins/hunspell/hunspelldict.cpp
--- sonnet-5.11.0/src/plugins/hunspell/hunspelldict.cpp	2015-06-06 21:32:56.000000000 +0000
+++ sonnet-5.11.0.tpg/src/plugins/hunspell/hunspelldict.cpp	2015-06-14 15:13:50.630424723 +0000
@@ -31,9 +31,9 @@
     : SpellerPlugin(lang), m_speller(0)
 {
     qCDebug(SONNET_HUNSPELL) << " HunspellDict::HunspellDict( const QString& lang ):" << lang;
-    QString dic = QStringLiteral("/usr/share/myspell/dicts/%1.dic").arg(lang);
+    QString dic = QStringLiteral("/usr/share/dict/ooo/%1.dic").arg(lang);
     if (QFileInfo(dic).exists()) {
-        m_speller = new Hunspell(QStringLiteral("/usr/share/myspell/dicts/%1.aff").arg(lang).toUtf8().constData(), dic.toUtf8().constData());
+        m_speller = new Hunspell(QStringLiteral("/usr/share/dict/ooo/%1.aff").arg(lang).toUtf8().constData(), dic.toUtf8().constData());
     } else {
         m_speller = 0;
     }
