diff -Naur sonnet-5.20.0/src/plugins/hunspell/hunspelldict.h sonnet-5.20.0.tpg/src/plugins/hunspell/hunspelldict.h
--- sonnet-5.20.0/src/plugins/hunspell/hunspelldict.h	2016-03-05 19:15:12.000000000 +0000
+++ sonnet-5.20.0.tpg/src/plugins/hunspell/hunspelldict.h	2016-03-13 12:57:19.262161302 +0000
@@ -28,7 +28,7 @@
 #ifdef Q_OS_MAC
 #define HUNSPELL_MAIN_DICT_PATH "/System/Library/Spelling"
 #else
-#define HUNSPELL_MAIN_DICT_PATH "/usr/share/hunspell"
+#define HUNSPELL_MAIN_DICT_PATH "/usr/share/dict/ooo"
 #endif
 #endif
 
